env/common/files/galaxy/dynamic_rules/test/unit/test_mdw.py

```diff
@@ -186,7 +198,25 @@
     {"ref_size": 1024 ** 4,
      "tool": tool_tophat2_indexed,
      "sbatch_node": "roundup",
-     "resource_params": None,
+     "resource_params": {"tacc_compute_resource": "stampede_normal"},
+     "return_submit_native_specification": "--partition=normal --nodes=1 --cpus-per-task=16 --time=48:00:00 --account=TG-MCB140147",
+     "return_destination_id": "stampede_normal"},
+    {"ref_size": 1024 ** 4,
+     "tool": tool_tophat2_history,
+     "sbatch_node": "roundup",
+     "resource_params": {"tacc_compute_resource": "stampede_normal"},
+     "return_nativeSpecification": "--partition=multi --nodes=1 --cpus-per-task=6 --time=36:00:00",
+     "return_destination_id": "slurm_multi"},
+    {"ref_size": 1024 ** 4,
+     "tool": tool_hisat2_indexed,
+     "sbatch_node": "roundup",
+     "resource_params": {"tacc_compute_resource": "stampede_normal"},
+     "return_submit_native_specification": "--partition=normal --nodes=1 --cpus-per-task=16 --time=48:00:00 --account=TG-MCB140147",
+     "return_destination_id": "stampede_normal"},
+    {"ref_size": 1024 ** 4,
+     "tool": tool_hisat2_history,
+     "sbatch_node": "roundup",
+     "resource_params": {"tacc_compute_resource": "stampede_normal"},
      "return_nativeSpecification": "--partition=multi --nodes=1 --cpus-per-task=6 --time=36:00:00",
      "return_destination_id": "slurm_multi"},
```

Comment:
These seem backwards to me? If it's indexed it shouldn't return stampede, if it's history it should?

Reply:
for reference: we figured out the actual rule was reversed (thank you unit test!) 
cc52b8b should fix it